src/index.md

```diff
@@ -16,7 +16,7 @@ Iroha 2 is a minimal code base. We take great care to vet our dependencies, and
 
 We have designed our own actor framework, to both keep Iroha 2 neat and also avoid many of the pitfalls common to asynchronous networking. Consequently, Iroha 2 is smaller and more reliable than anything written using `actix`. It's very hard to make Iroha 2 hang or deadlock, you can trust us, we tried. As Iroha 2 is written in Rust, and uses the language's features to great effect. We avoid `panics` and `unsafe` code. Our code base uses `enum` types extensively, as both a means of type erasure and boxing.
 
-Iroha 2 is also more flexible than the original Iroha, and it is highly modular in its design. It should be possible to add/remove features based on the particular use-case.  If you want to be extremely fast, and work on a very small embedded piece of hardware, just compile Iroha 2 without the `expensive-metrics` feature. Don't use telemetry at all? Remove it. Need to have roles in your blockchain? Enable the `roles` feature. Want a permissioned blockchain? or maybe a permissionless one? You got it all in one neat little package called **Iroha 2**.
```

Comment:
For the record, two spaces after a full stop is an old-school best practice, especially in monospace plain-text formats such as LaTeX, Org, RST, Markdown etc.